dubbo-dependencies-bom/pom.xml

```diff
@@ -571,6 +574,19 @@
                 <artifactId>testcontainers</artifactId>
                 <version>${test_container_version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>javax.annotation</groupId>
+                <artifactId>javax.annotation-api</artifactId>
+                <version>${javax_annotation_api_version}</version>
+            </dependency>
```

Comment:
Why do we need this declaration? I don't find the place where it was referenced.

Reply:
yes. i'll remove..i'm just merge this from rpc-native-thrift-for-apache-dubbo  to here. it's  my fault.btw i find  the thirft version of rpc-native-thrift and rpc-thrift is not same. should i make it same or declare 0.11 just in rpc-native-thrift module ??